core/pkg/server/stream_init.go

```diff
@@ -62,6 +80,7 @@ func NewGraphQLClient(
 		NonRetryTimeout: api.DefaultNonRetryTimeout,
 		ExtraHeaders:    graphqlHeaders,
 		NetworkPeeker:   peeker,
+		Proxy:           ProxyFn(settings.Proto.GetXProxies().GetValue()),
```

Comment:
instead of accessing via `settings.Proto`, can you document the `_proxies` setting in `wandb_settings.proto` and add a corresponding getter method in `settings.go`?

the documentation should explain the expected format and how to use the setting